tests/test_helpers.py

```diff
@@ -21,37 +21,9 @@ def _check_ssl_dir(self):
         self.assertTrue(keyfile.exists(), str(keyfile))
         return cafile, certfile, keyfile
 
-    @pytest.mark.skipif(sys.version_info >= (3, 4),
```

Comment:
How does this relate to 3.3?

Reply:
description was wrong, this test was 3.3 specific, as indicated by `skipif(sys.version_info >= (3, 4)`